### ISSUE TITLE
【0.5.0-rc2】generate bug: object with "id" property

### ISSUE THREAD

**Issue by mansonchor** [json-schema-faker/json-schema-faker#273]
gist: http://json-schema-faker.js.org/#gist/9f00854e409241777914afc55a58bc93
version: **0.5.0-rc2**

```js
{
    "type": "object",
    "properties": {
        "id": {
            "type": "string"
        }
    },
    "required": [
        "id"
    ]
}
```

expect

```js
{
  id: "asfsaf"
}
```

but get sth wrong data
```js
{
  "elit64": 41162889
}
```

but in **0.5.0-rc1**, that's ok~~

so is **0.5.0-rc2** bring the bug

**Comment by sirchris** [comment]
I've got the same problem in 0.5.0-rc1, weird cause yesterday, even today morning everything worked well :(

**Comment by mansonchor** [comment]
`0.5.0-rc3` release, but still not fixed yet

**Comment by dolphub** [comment]
+1 Please get this fixed.

**Comment by justVitalius** [comment]
+1 Please get this fixed.

**Comment by lolochka** [comment]
+1

**Comment by pateketrueke** [comment]
Sorry for the lag, this will be fixed on `0.5.0-rc4` - it was my fault.

**Comment by pateketrueke** [comment]
Ok, I figured out the correct settings for using `json-schema-ref-parser` and `deref` together to also make circular-refs work. 

Closing this in order to release `0.5.0-rc5` :beers:
